android/lib/src/main/java/com/swmansion/gesturehandler/GestureHandler.kt

```diff
@@ -274,11 +298,112 @@ open class GestureHandler<ConcreteGestureHandlerT : GestureHandler<ConcreteGestu
     }
   }
 
+  fun updatePointerData(event: MotionEvent) {
+    trackedPointers?.let { trackedPointers ->
+      pointerEventPayload = null
+
+      val offsetX = event.rawX - event.x
+      val offsetY = event.rawY - event.y
+
+      if (event.actionMasked == MotionEvent.ACTION_DOWN || event.actionMasked == MotionEvent.ACTION_POINTER_DOWN) {
+        pointerEventType = RNGestureHandlerPointerEvent.EVENT_POINTER_DOWN
+        val pointerId = event.getPointerId(event.actionIndex)
+
+        trackedPointers[pointerId] = PointerData(
```

Comment:
You're right, but in order to be consistent with how it's implemented on iOS, where only the new pointers are in the `DOWN` event, would sending two events in this case be a good solution? I.e. send `DOWN` up with just the data about the new pointer and then send the `MOVE` event with data about all pointers.

This would also apply to the `UP` event, right?

Reply:
I don't think it matters that much if we send one or two events at the moment when new finger is placed on the screen. What's more important is for this to work consistently across platforms. So if that's how you want it I'm fine with it so long it works the same on Android and iOS

Reply:
Ok, I made it try sending two events for now: one down/up event and a move event with data about other pointers in https://github.com/software-mansion/react-native-gesture-handler/pull/1673/commits/06ae23ae8e9c6b42b728d43f551e9556daa41bc1.